File: RollingPictures-Backend/api-module/src/main/java/com/ssafy/api/dto/res/ChannelResDTO.java
package com.ssafy.api.dto.res;

import com.ssafy.core.code.YNCode;
import lombok.*;

import java.util.List;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ChannelResDTO {
    private long id;

    private String code;

    private String title;

    private YNCode isPublic;

    private int maxPeopleCnt;

    private int curPeopleCnt;

    private List<UserInfoResDTO> users;
}

File: RollingPictures-Android/app/src/main/java/com/firechicken/rollingpictures/dto/ChannelResDTO.kt
package com.firechicken.rollingpictures.dto

data class ChannelResDTO(

    var id: Long,
    var code: String,
    var title: String,
    var isPublic: String,
    var maxPeopleCnt: Int,
    var curPeopleCnt: Int,
    var users: MutableList<UserInfoResDTO>
)
